src/core/network.js

```diff
@@ -75,6 +75,21 @@ var NetworkManager = (function NetworkManagerClosure() {
     return array.buffer;
   }
 
+//#if (GENERIC || FIREFOX || B2G || MOZCENTRAL)
```

Comment:
I have added another commit to the PR that completely eliminates this extra variable in Firefox. Could you take a look?